libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -435,16 +437,16 @@ uint32_t AP_Frsky_Telem::get_next_msg_chunk(void)
 {
     if (_msg_chunk.repeats == 0) {
         _msg_chunk.chunk = 0;
-        uint8_t character = _msg.data[_msg.sent_idx].text[_msg_chunk.char_index++];
+        uint8_t character = _statustext_queue[0]->text[_msg_chunk.char_index++];
```

Comment:
I see that, but what happens when this function gets reused? The check should be in the function, it's not wise to have other functions sanity check contents of other functions. That's a future bug in the making..